pyproject.toml

```diff
@@ -1,11 +1,46 @@
-[build-system]
+[project]
 name = "ElectionLeaflets"
-version = "3.0-alpha"
-requires = [
-    "setuptools >= 35.0.2",
-    "setuptools_scm >= 2.0.0, <3"
+version = "0.1.0"
+description = " 📮 A Django port of the original PHP code from electionleaflets.org "
```

Comment:
Minor point, but I reckon we can probably have a bit more faith in ourselves :smile: 
Suggestion: "An online archive of political leaflets"

Reply:
Woah